### PR TITLE
{tools}[dummy/] Lucene-Geo-Gazetteer v20170718

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e1f6a7aae91066adb494108564e253a8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/34411f37645a0c11a27f16717fb4c8e2 for a full test report.

File: easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'PackedBinary'
+
+name = 'Lucene-Geo-Gazetteer'
+version = '20170718' 
+
+homepage = 'https://github.com/chrismattmann/lucene-geo-gazetteer'
+description = """A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library
+ to create a searchable gazetter."""
```
Comment (verdurin) [inline easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb]: @boegel Please use the same spelling of gazetteer within the same sentence...

File: easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/chrismattmann/lucene-geo-gazetteer/archive/']
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': '6c38272.tar.gz'}]
+checksums = ['2ed355bfdcf865d815df9dc52ee747eeb8a15d065ef159ebf933ad36e1555ad4']
+
+builddependencies = [('Maven', '3.5.0')]
+dependencies = [('Java', '')]
```
Comment (verdurin) [inline easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb]: @boegel This is odd - does it really only need *any* version of `Java`?

File: easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb]: Copy-paste from the README at https://github.com/chrismattmann/lucene-geo-gazetteer, but I guess it's a typo... I'll fix it, also sent them a PR: https://github.com/chrismattmann/lucene-geo-gazetteer/pull/26

File: easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb]: Hah! This is obviously wrong, it should specify a particular version of Java, will fix.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6cad9c02e2626d75aa43a0bfb137dece for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2f9a6b8e5fde912f7504a1783cabc907 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2c990dd90ccf16293d4fa3f99e7218e6 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**